webcompat/form.py

```diff
@@ -48,32 +52,42 @@
 username_message = u'A valid username must be {0} characters long'.format(
     random.randrange(0, 99))
 
+desc_label = 'Please describe what was wrong <span class="wc-Form-required">' \
+    '*</span>'
+desc_message = u'An issue description is required.'
+
 problem_label = (u'What seems to be the trouble?',
                  '<span class="wc-Form-required">*</span>')
 url_label = u'Site URL <span class="wc-Form-required">*</span>'
 
-desc_default = u'''1. Navigate to: Site URL
-2. …
+steps_default = u'''1. I've tried to log in.
+2. I've filled out the form details.
+3. I clicked on the submit button.
+4. Nothing happened.
 
-Expected Behavior:
-
-Actual Behavior:
 '''
 
 
 class IssueForm(FlaskForm):
     '''Define form fields and validation for our bug reporting form.'''
     url = StringField(url_label,
                       [InputRequired(message=url_message)])
-    browser = StringField(u'Browser / Version', [Optional()])
+    browser = StringField(u'Is this information correct?', [Optional()])
     os = StringField(u'Operating System', [Optional()])
     username = StringField(u'Username',
                            [Length(max=0, message=username_message)])
-    description = TextAreaField(u'Give more details', [Optional()],
-                                default=desc_default)
+    description = StringField(desc_label,
+                              [InputRequired(message=desc_message)])
+
+    steps_reproduce = TextAreaField(u'How did you get there?', [Optional()],
+                                    default=steps_default)
     problem_category = RadioField(problem_label,
                                   [InputRequired(message=radio_message)],
                                   choices=problem_choices)
+    browser_test = RadioField(problem_label,
```

Comment:
Oh wait, I think I know why we didn't do that. It's because that will generate the `<label></label>` and we need content to go inside. Since we're not using the value of `problem_label` in form.py, maybe we should just leave a comment saying it's unused, but `RadioField` does need a label argument.